src/select-list-view.ts

```diff
@@ -474,7 +508,7 @@ class ListItemView {
 
   scrollIntoViewIfNeeded () {
     if (this.selected) {
-      this.element.scrollIntoViewIfNeeded(false)
+      this.element.scrollIntoView(false)
```

Comment:
This is a non-standard method, and it does not exist in the TypeScript types.
https://developer.mozilla.org/en-US/docs/Web/API/Element/scrollIntoViewIfNeeded

Reply:
Ahh...right. No, this still needs to be scrollIntoViewIfNeeded to avoid unexpected scrolling. Can this be ts-ignored instead of declaring the type? :D